GenderPayGap.WebUI/Models/Account/ChangePasswordNewViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+using GovUkDesignSystem;
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+
+namespace GenderPayGap.WebUI.Models.Account
+{
+    public class ChangePasswordNewViewModel : GovUkViewModel
+    {
+
+        [GovUkValidateRequired(ErrorMessageIfMissing = "Please enter your current password")]
+        public string CurrentPassword { get; set; }
+        
+        [GovUkValidateRequired(ErrorMessageIfMissing = "Please enter a new password")]
+        public string NewPassword { get; set; }

```

Comment:
As mentioned, we can use the `[GpgPasswordValidation]` attribute here to do most of our validation

Reply:
Does this cover our basic password expectations - one upper case, one lower case, one number and >8 characters? I hadn't seen this - very clever!